packages/core/src/components/Pressable.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import {
+  Pressable as NativePressable,
+  PressableProps,
+  ViewStyle,
+} from "react-native";
+
+type Props = {
+  disabled?: boolean;
+  children: React.ReactNode;
+  style?: ViewStyle;
+  onPress?: () => void;
+  onLongPress?: () => void;
+  delayLongPress?: number;
+  hitSlop?: number;
+  activeOpacity?: number;
+  disabledOpacity?: number;
```

Comment:
Can we omit the ones of these which are already covered by `NativePressable`? I think for example `onPress`, `onLongPress`, `children` etc. This would allow us to see at a glance what this wrapper was adding, which is good info especially if we're going to phase out jigsaw eventually.

Reply:
I tried to slim it down and got to Style, activeOpacity, and disabledopacity in the types.